src/std/misc/lru.ss

```diff
@@ -3,15 +3,17 @@
 ;;; LRU cache
 
 (import :std/error
-        :std/misc/list
+        :std/contract
         :std/generic
-        :std/iter)
+        :std/iter
+        :std/misc/list)
 (export make-lru-cache lru-cache? lru-cache
         lru-cache-ref lru-cache-get lru-cache-put! lru-cache-remove!
         lru-cache-size (rename: lru-cache-cap lru-cache-capacity)
         lru-cache-flush! lru-cache-for-each lru-cache-walk
         lru-cache-fold lru-cache-foldr lru-cache->list
         in-lru-cache in-lru-cache-keys in-lru-cache-values)
+(declare (not safe))
```

Comment:
Intentional?

Reply:
yes, we have `:` checks at entry so we can do it.